docs/standards/standard-detection.md

```diff
@@ -17,15 +17,68 @@ There are 2 types of standards at **LUKSO**:
 
 - **Metadata Standards**: Where we standardize a set of ERC725Y keys. i.e: [LSP3-UniversalProfile-Metadata](./universal-profile/03-lsp3-universal-profile-metadata.md), [LSP4-DigitalAsset-Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md), [LSP10ReceivedVaults](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-10-ReceivedVaults.md), etc ..
 
-## Difference between InterfaceId VS SupportedStandard Key
+![Interface and metadata standards](../../static/img/standard-detection.jpeg)
+
+These 2 types of standards are fundamental for interacting with smart contracts on the LUKSO blockchain.
+
+**Interface Standard** enables us to know which functions (and their parameters) can be called on a smart contract. On the other hand, **Metadata Standard** gives information about the data set by default on the contract, and which keys can be queried to retrieve such data.
+
+## Interface Detection
+
+:::success Tip
+
+For a full list of `interfaceId`s, see the section **[Contracts Implementation > Interface Ids](./smart-contracts/interface-ids)**
+
+:::
+
+> This covers how to detect if a contract implements a specific interface.
 
+We can verify if a contract implements a specific set of functions (= an **interface**) using an [`interfaceId`](./smart-contracts/interface-ids).
 
-Checking the **InterfaceId** for a contract using [ERC165 standard](https://eips.ethereum.org/EIPS/eip-165) is equal for checking if the contract is implementing a set of functions. As for checking for the **SupportedStandards:XYZ** ERC725Y key, it's equal for checking if the contract is implementing **XYZ** related ERC725Y Keys.
+This can be done using the function `supportsInterface(interfaceId)`, passing the `bytes4` `interfaceId` as a parameter.
+
+Calling this function will return **TRUE** if the contract implements this specific `interfaceId`, **FALSE** otherwise.
 
 ### Example
 
-![Interface and metadata standards](../../static/img/standard-detection.jpeg)
+A **UniversalProfile** is a contract based on **[LSP0 - ERC725Account](./universal-profile/01-lsp0-erc725account.md)** (LSP0). This means that the contract **SHOULD** implement the functions defined in the [LSP0 - ERC725Account interface](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md#interface-cheat-sheet): `getData(...)`, `setData(...)`, `execute(...)`, `universalReceiver(...)`, `isValidSignature(...)` .
```

Comment:
```suggestion
A **UniversalProfile** is a contract based on **[LSP0 - ERC725Account](./universal-profile/01-lsp0-erc725account.md)** (LSP0). This means that the contract **SHOULD** implement the functions defined in the [LSP0 - ERC725Account interface](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md#interface-cheat-sheet): `getData(...)`, `setData(...)`, `execute(...)`, `universalReceiver(...)`, `isValidSignature(...)`.
```

`owner`, `transferOwnership`... are also in the cheat sheet - should they appear in this list?
Should it be the element of the cheat sheet or the element you listed?

Maybe not listing these function is better because you won't have to maintain this list if these functions change?

Reply:
Agree with this point.